sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/math.scala

```diff
@@ -255,3 +257,39 @@ case class Pow(left: Expression, right: Expression)
       """
   }
 }
+
+case class Round(left: Expression, right: Expression)
```

Comment:
We can simply override the `def left` and `def right` methods. The `left` or `right` here is meaningless for `Round`. 
However, the biggest concern here is, seems the `Round` in Hive only accept the constant value for `scala`, which mean the `Round` is a `UnaryExpression`, and the second argument will be just `Int`.